spec/query_spec.cr

```diff
@@ -757,4 +757,18 @@ describe Avram::Query do
       end
     end
   end
+
+  describe "#reset_limit" do
+    it "resets the limit to nil so we can use aggregate functions again" do
```

Comment:
I think it’d be good to remove “so we can use aggregate functions” because it can be used in other ways. 

If you want to test that select count still works then I think a separate spec might be clearer “limits and offsets can be reset so aggregates still function” but I think that may nit be necessary. It’s up to you

Reply:
True. Really we just need to test the the limit and offset have a value before the reset, and no value after. I'll update the specs.